### PR TITLE
Add Opportunist data

### PR DESCRIPTION
**Pull request by HaraldJoerg**
The higher career task are most probably illegal, too, but I have no data yet.

### CODE REVIEW

**Comment by Perleone** [comment]
@HaraldJoerg Please fix the conflicts first.

**Comment by HaraldJoerg** [comment]
Conflicts resolved. I should have written a mail after submitting the PR.

File: career.md [hunk 1]
```diff
@@ -31,34 +31,42 @@ Social, Stamina
...
+Advantages:
+
+* Many career tasks can be performed anywhere, or at least on any station
+
 Disadvantages:
 
-* Very narrow focus on two stats, especially in the lower range
-* Presumably many illegal career tasks
+* Many illegal career tasks
+* Only three stats, the third (intelligence) chiming in rather late
```
Comment (moritz) [inline career.md]: Did you mean "Only *two* stats"?

**Review by Perleone — APPROVED** [review]

**Comment by moritz** [comment]
Thanks @HaraldJoerg, and sorry for causing the merge trouble in the first place.

**Comment by HaraldJoerg** [comment]
No problem. 'Twas easy, and important, 'cause I missed the most obvious disadvantage of criminals' tasks: They are likely to be illegal.  Comes with the career, I guess.

**Pull request merged**